x-pack/metricbeat/module/aws/sns/manifest.yml

```diff
@@ -0,0 +1,12 @@
+default: true
+input:
+  module: aws
+  metricset: cloudwatch
+  defaults:
+    metrics:
+      - namespace: AWS/SNS
+        statistic: ["Average"]
+        name: ["PublishSize", "SMSSuccessRate"]
+      - namespace: AWS/SNS
+        statistic: ["Maximum"]
+        name: ["NumberOfMessagesPublished", "NumberOfNotificationsDelivered", "NumberOfNotificationsFailed", "NumberOfNotificationsFilteredOut", "NumberOfNotificationsFilteredOut-InvalidAttributes", "NumberOfNotificationsFilteredOut-NoMessageAttributes", "NumberOfNotificationsRedrivenToDlq", "NumberOfNotificationsFailedToRedriveToDlq", "SMSMonthToDateSpentUSD"]
```

Comment:
Sorry for not noticing this earlier but maybe we could change the format here for readability, since there are many array items, like:
```yml
name:
  - NumberOfMessagesPublished
  - NumberOfNotificationsDelivered
  - NumberOfNotificationsFailed
  - NumberOfNotificationsFilteredOut
  ...
```

wdyt?

Reply:
Fixed.